build/download_kubectl_binary.sh

```diff
@@ -5,22 +5,6 @@ download_kubectl_binary(){
   local KUBECTL_VERSION=$3
   local KUBECTL_BIN_NAME="kubectl"
 
-  if [ "$PLATFORM" = "win" ]; then
```

Comment:
Simple question but I'm assuming we're now using the go OS and ARCH references - therefore there is no need for a translation anymore, right?

Reply:
Correct.